src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -396,16 +397,18 @@ const AvatarCropModal = (props) => {
                                 src={Expensicons.Zoom}
                                 fill={themeColors.icons}
                             />
-                            <Pressable
+                            <PressableWithoutFeedback
                                 style={[styles.mh5, styles.flex1]}
                                 onLayout={initializeSliderContainer}
                                 onPressIn={(e) => runOnUI(sliderOnPress)(e.nativeEvent.locationX)}
+                                accessibilityLabel="slider"
+                                accessibilityRole="adjustable"
```

Comment:
We are adding a CONST for accessibilityRole here - https://github.com/Expensify/App/pull/20251/files#diff-9d6b0cc89c7cab66ac4042ac0549bfc66ddc9e7a182b6b5dbbf7f687328f10b5R2451

We should incorporate these changes as well before we push

Reply:
@abdulrahuman5196  I think there are some problems with merging PR that adds CONSTs. In other similar PRs we agreed create another PR that will change accessibilityLabels everywhere to use CONSTS, so e can try to merge this PR without it